src/app/shared/components/image-carousel/image-carousel.component.scss

```diff
@@ -1,8 +1,60 @@
-.image-container {
-  height: 300px;
+$image-height: 400px;
+$dots-height: 24px;
+$icon-size: 24px;
+$button-size: 50px;
+$button-bg-color-hover: #3849f9;
+$button-padding: 50px;
 
-  .image {
-    object-fit: contain;
-    width: 100%;
+.image {
+  object-fit: contain;
+  width: 100%;
+  height: $image-height;
+  position: relative;
+}
+
+::ng-deep {
+  .owl-prev,
+  .owl-next {
+    position: absolute;
+    top: calc(50% - $dots-height);
+
+    width: $button-size;
+    height: $button-size;
+    transform: translateY(-50%) !important;
+    padding: 0 !important;
+    border-radius: 50% !important;
+
+    display: flex !important;
```

Comment:
Should we definitely use `!important` here? I believe there's possibility to apply required properties using selector with bigger specificity.
Also, put `::ng-deep` under some selector for carousel or at least `:host` - because these styles then will become global.